scripts/blueprints/stack-arch.yaml

```diff
@@ -0,0 +1,25 @@
+selector:
+  sw.os: 1
+  sw.stack: 1
+  sw.stack-variant: 1
+  arch.sw: 1
+output:
+  filename: Dockerfile
+  type: sw.docker-image.stack-arch.{{this.children.sw.stack.slug}}
+  slug: "{{this.children.arch.sw.slug}}-{{this.children.sw.os.slug}}-{{this.children.sw.stack.slug}}"
```

Comment:
The slug is meant to be a unique identifier for the contract. Right now two images with different variants would contain the same slug, which feels wrong. I understand we are using the `slug` for the `FROM` part of the image, but maybe we can include variants in the slug, and have another contract property (i.e. `imageName`) that doesn't include it? 

Not an urgent thing right now, but it will make it easier when we start pushing these contracts to Jellyfish.

Reply:
@jviotti, I added `imageName` property in the blueprints and updated the slug so it will have stack versions and variants included. Hence the slug will always be unique.